pages/app.vue

```diff
@@ -1009,6 +1009,8 @@ useSeoMeta({
     margin: 0 auto;
     justify-content: center;
     mask-image: none;
+    position: relative;
+    z-index: 20;
```

Comment:
20 seems overkill, 1 should do, right? 20 may appear on top of other elements that are attempting to cover the screen.